packages/nimble/inst/tests/test-size.R

```diff
@@ -28,21 +28,23 @@ testsScalar <- list(
     list(name = 'scalar stochastic, parameter expression', expectPass = TRUE,
          expr = quote({y ~ dnorm(mu1 + mu2, sd = sig)}), 
          inits = list(mu1 = 0, mu2 = 0, sig = 1) ),
-    list(name = 'scalar stochastic, parameter expression non-scalar, no indices', expectPass = FALSE, expectPassWithConst = TRUE,
-         knownProblem = TRUE,
+    list(name = 'scalar stochastic, parameter expression non-scalar, no indices',
+         expectPass = FALSE, expectPassWithConst = TRUE,
+         knownProblem = TRUE, knownProblemWithConst = FALSE,
          expr = quote({y ~ dnorm(mu1%*%mu2, sd = sig)}), 
          inits = list(mu1 = vec2, mu2 = vec2, sig = 1) ),
     
     list(name = 'scalar stochastic, parameter expression non-scalar, RHS index',
-         expectPass = FALSE,
-         knownProblem = TRUE,
+         expectPass = FALSE, expectPassWithConst = FALSE,
```

Comment:
What this means is that
- for variables: nimble should FAIL but does not, and this is a known problem.
- for constants: nimble should FAIL and does, so there is no problem
Does that make sense?